docs/user_manual/source/fencei.rst

```diff
@@ -0,0 +1,10 @@
+.. _fencei:
+
+Fence.i external handshake
+==========================
+|corev| includes an external handshake that will be exercised upon execution of the fence.i instruction.
+The handshake is composed of the signals ``fencei_flush_req_o`` and ``fencei_flush_ack_i``.
```

Comment:
The handshake can for example be used to flush an externally connected cache.

Reply:
Updated.